doc/user/content/free-trial-faqs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Free trial FAQs"
+description: "Answers to frequency asked questions about Materialize free trials"
+menu:
+  main:
+    parent: "quickstarts"
+    weight: 15
+---
+
+When you [sign up for Materialize](https://materialize.com/register/), you get a free trial account so you can explore the product without worrying about initial costs. This page answers some frequently asked questions about free trials.
+
+## What's included in a free trial?
+
+In Materialize, [cluster replicas](/overview/key-concepts/#cluster-replicas) are the physical resources for doing computational work. Cluster replicas come in various sizes, and each size corresponds to a specific number of [compute credits per hour](https://materialize.com/pricing/).
+
+During your free trial, your rate of credit consumption in a given region cannot exceed 4 credits per hour at any point in time. This limit should accommodate most trial scenarios. If you need additional resources during your trial, please contact sales@materialize.com.
```

Comment:
In line with [this comment](https://materializeinc.slack.com/archives/C0538L6HFJM/p1682457957124089?thread_ts=1682456986.458229&cid=C0538L6HFJM), it'd be nice to provide two options here and soften the mention to sales: link to the community Slack + link to a "Talk with your team" form (_i.e._ soften the mention to sales).

Reply:
Made some changes based on a Slack discussion.